Library/Homebrew/utils/shell.rb

```diff
@@ -0,0 +1,64 @@
+require 'shellwords'
```

Comment:
My mistake; I suggested `shellwords` without checking if it was available for 1.8.7 :see_no_evil: